site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -150,6 +150,8 @@ The default queueing strategy is `BestEffortFIFO`.
 ClusterQueues can be grouped in _cohorts_. ClusterQueues that belong to the
 same cohort can borrow unused quota from each other.
 
+![Cohort](/images/cluster-queue.svg)
```

Comment:
This is not so much about cohort, but all of the ClusterQueue concept. Put it before the first subtitle